ExpenseTracker/ExpenseTracker/Views/Notifications/Index.cshtml

```diff
@@ -161,4 +174,63 @@
         background-color: #ffffff;
         border-color: #d1d5da;
     }
-</style>
\ No newline at end of file
+
+    /* Popup Modal Styles */
```

Comment:
I think this **Popup modal styles** is not necessary, because we have styles for popups

Reply:
ok i fixed it